docs/index.md

```diff
@@ -111,6 +111,7 @@ options for deployment:
 * [GraphX](graphx-programming-guide.html): processing graphs 
 * [SparkR](sparkr.html): processing data with Spark in R
 * [PySpark](api/python/getting_started/index.html): processing data with Spark in Python
+* [Spark SQL CLI](sql-distributed-sql-engine-spark-sql-cli.html): process data use SQL with command line interface
```

Comment:
Probably "processing data with SQL on the command line"

Reply:
Done